source/Playnite/Database/GameDatabase.cs

```diff
@@ -1310,6 +1313,35 @@ bool updateCompletionStatus(Game game, CompletionStatusSettings settings)
             }
         }
 
+        public List<FilterPreset> GetSortedFilterPresets()
+        {
+            var filterPresetsSettings = (FilterPresets as FilterPresetsCollection).GetSettings();
+            var sortingOrder = filterPresetsSettings.SortingOrder;
+            var sortingDict = sortingOrder.ToDictionary(x => x, x => sortingOrder.IndexOf(x));

```

Comment:
I'm not sure if this dictionary is necessary. You can just use IndexOf and Contains on the original SortingOrder list to generate those other lists.

Reply:
I did it for performance reasons but yeah that's overkill to be honest. Done in https://github.com/JosefNemec/Playnite/pull/3258/commits/0c4a6c036edd6976cc0e4aad2bd0a27b423777bf